app/main.py

```diff
@@ -1,2 +1,68 @@
 class Dictionary:
-    pass
+
+    def __init__(self):
+        self.key = None
+        self.value = None
+        self.capacity = 8
+        self.threshold = 5
```

Comment:
What is it?

Reply:
the same question

Reply:
I need self.capacity, self.threshold, self.length values to use them. Can I create them as a class atributes?

Reply:
yeap

Reply:
self.threshold - what about it? Why do you use her?

Reply:
I agree. I don`t need it. But what about self.length and self.capacity = 8? Why creating this attributes for instance is a bad idea?